seesaw/seesaw.go

```diff
@@ -67,7 +69,18 @@ func (d *Device) ReadHardwareID() (byte, error) {
 		return hwid, nil
 	}
 
-	return 0, errors.New("unknown hardware ID: " + strconv.Itoa(int(hwid)))
+	return 0, fmt.Errorf("unknown hardware ID: %0X", hwid)
+}
+
+// ReadVersion reads the version bytes from the device (undocumented in the datasheet)
+func (d *Device) ReadVersion() (uint32, error) {
+
+	buf := make([]byte, 4)
+	err := d.Read(ModuleStatusBase, FunctionStatusVersion, buf, defaultDelay)
+	if err != nil {
+		return 0, err
+	}
+	return (uint32(buf[0]) << 24) | (uint32(buf[1]) << 16) | (uint32(buf[2]) << 8) | uint32(buf[3]), nil
```

Comment:
You have made some changes to the generic seesaw support (seesaw/seesaw.go), perhaps this should be included in #565?